CONTRIBUTING.rst

```diff
@@ -18,6 +18,16 @@ will automatically be deployed to Heroku. This process takes less than 5 minutes
 If after 10 minutes you did not see the changes reflected, please ping one
 of the collaborators listed below.
 
+To enable Automatic deployment:
+
+- On the Heroku dashboard for bedevere, choose the "Deploy" tab.
+- Scroll down to the "Automatic deploys" section
+- Enter the name of the branch to be deployed (in this case: ``main``)
+- Check the "Wait for CI to pass before deploy" button
+- Press the "Enable automatic deploys" button.
+
+Once done, merging a PR against the ``main`` branch will trigger a new deployment.
```

Comment:
I checked in the settings of the repo and I see there is a webhook linked to heroku.  If this is the mechanism that triggers the deployment I think it should be mentioned, since it's not clear if it's triggered by a webhook, a GitHub action, by Heroku polling the repo or something else.